boards/arm/nrf52_sparkfun/Kconfig.board

```diff
@@ -0,0 +1,10 @@
+# Kconfig - Sparkfun nRF52832 breakout board configuration
+#
+# Copyright (c) 2016 Linaro Limited.
```

Comment:
I Linaro the right copyright owner for this file?
Similar question to other private filed this board.

Reply:
This is a lightly modified blenano2 port, Linaro wrote more of this than me. If there's an official correct answer, I'll change this to what ever is recommend.

Reply:
Recommendation is to add to the list of copyright holders your name or your company's, whichever applies. Like here: https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/common/log.c#L4